examples/LabeledMerge.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { Range, getTrackBackground } from '../src/index';
+import { useThumbOverlap } from '../src/utils';
+
+const STEP = 0.1;
+const MIN = 0;
+const MAX = 100;
+const COLORS = ['#0C2960', '#276EF1', '#9CBCF8', '#ccc'];
+const THUMB_SIZE = 42;
+
+const ThumbLabel = ({
+  rangeRef,
+  values,
+  index,
+}: {
+  rangeRef: Range | null;
+  values: number[];
+  index: number;
+}) => {
+  const [labelValue, style] = useThumbOverlap(rangeRef, values, index);
+  return (
+    <div
+      style={{
```

Comment:
in puppeteer, we want to be able to target these elements, so let's add an identifier:

```
data-label={index}
```